### ISSUE TITLE
Building a 4-class classifier with a 2-qubit circuit

### ISSUE THREAD

**Issue by DarthMalloc** [mit-han-lab/torchquantum#69]
I am wondering about a possible alternative approach for using quantum measurement results to perform a 4-class classification task. Basically I would like to be able to carry out the classification with a 2-qubit architecture. Each class would be assigned to one of the computational basis states among the possible measurement results. I would perform the appropriate operations on the circuit, measure the state with a certain number of shots, and create a 4-element tensor each element corresponding to the number of shots producing each measurement results, so you have one value for the number of "00" results and so on. I would then take a softmax of those values to get the classification result. What I am wondering is whether such a method cam work with pytorch quantum. If so, since I don't have a lot of experience with pytorch quantum,  I would appreciate if someone would be willing to show me how to make sure that I am doing this without accidentally breaking the computational graph of pytorch and preventing the parameters of my model from being updated.

**Comment by Hanrui-Wang** [comment]
Sorry for my late reply, I have constructed an example circuit to do MNIST 4-classification with 2 qubits. Let me know if you have any questions!

https://github.com/mit-han-lab/torchquantum/blob/main/examples/simple_mnist/mnist_2qubit_4class.py
